src/slurmctld_ops.py

```diff
@@ -0,0 +1,706 @@
+"""This module provides the SlurmManager."""
```

Comment:
Could you add an Apache-2.0 license header to this file? Something like the following should suffice:

```python
# Copyright 2024 Omnivector, LLC.
# See LICENSE file for licensing details.
```

This way we'll know that you are the original author of the file. I foresee it being around for a good while. We should likely have a discussion about what license headers should look like in our files. I'm an advocate for the full formal header, but not everyone uses PyCharm like me where you can set license templates :smile: